packages/api/src/graphql/blocks.ts

```diff
@@ -442,8 +442,18 @@ export const GraphQLLinkPageBreakBlock = new GraphQLObjectType<LinkPageBreakBloc
   name: 'LinkPageBreakBlock',
   fields: {
     text: {type: GraphQLString},
+    htmlText: {type: GraphQLString},
```

Comment:
We don't save pure HTML in the database. Please use RichTextNode[] => https://github.com/wepublish/wepublish/blob/master/packages/api/src/graphql/richText.ts

In the editor you can then also use the richTextEditor for formatting.

Reply:
also maybe a different name for the field since it's not clear what's the difference between `text` and `htmlText`

Reply:
- `text` is plaintext and needs to be untached (backwards compat)
- `htmlText` => is text which can container html (RichText is not matching case, must be saved as pure HTML in DB)

Reply:
in Short, rich Text us not suitable.

Reply:
(richText dont store pure HTML, we need pure HTML)